com.tntware.TntConnect.yml

```diff
@@ -0,0 +1,197 @@
+id: com.tntware.TntConnect
+branch: stable
+
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+base: org.winehq.Wine
+base-version: 'stable-21.08'
+tags:
+  - proprietary
+
+command: tntconnect.sh
+
+platform-extensions:
+  - org.freedesktop.Platform.Locale
+
+# Support 32-bit at build time
+sdk-extensions:
+  - org.freedesktop.Sdk.Compat.i386
+  - org.freedesktop.Sdk.Extension.toolchain-i386
+
+inherit-extensions:
+  - org.freedesktop.Platform.Compat.i386
+  - org.freedesktop.Platform.GL32
+  - org.freedesktop.Platform.VAAPI.Intel.i386
+  - org.winehq.Wine.DLLs
+  - org.winehq.Wine.gecko
+  - org.winehq.Wine.mono
+
+finish-args:
+  - --allow=devel # Without this "System Info" and "Gift Input" will crash
+  - --allow=multiarch
+  - --env=PATH=/app/bin:/app/lib/wine/bin:/app/lib/wine-32bit/bin:/app/lib/32bit/wine:/usr/lib/wine/bin:/usr/lib/wine-32bit/bin:/usr/bin:/usr/lib/32bit/bin
+  - --env=XDG_DATA_DIRS=/app/share:/usr/share:/usr/share/runtime/share:/run/host/share
+  - --env=WINEDEBUG=-all
+  - --env=WINEPREFIX=/var/data/wine
+  - --socket=x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --filesystem=xdg-documents
+  - --filesystem=~/.dropbox
+  - --filesystem=~/Dropbox
+  - --talk-name=org.freedesktop.portal.Documents
+
+cleanup:
+  - /man
+  - /share/man
+
+cleanup-platform:
+  - /include
+  - /share/aclocal
+  - /share/pkgconfig
+  - /lib/pkgconfig
+
+build-options:
+  cflags: -O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2
+  cxxflags: -O2 -g -fstack-protector-strong -D_FORTIFY_SOURCE=2
+  ldflags: -fstack-protector-strong -Wl,-z,relro,-z,now
+  prefix: /app
+  env:
+    V: '1'
```

Comment:
```suggestion
```

Given that you compile nothing... you don't need this, Also this is already set anyway.